docker-compose-dev.yaml

```diff
@@ -95,6 +95,9 @@ services:
     depends_on:
       badgerdoc-postgresql:
         condition: service_healthy
+    volumes:
+      - ./annotation/:/opt/annotation/
+    command: bash -c "alembic upgrade head && uvicorn annotation.main:app --host 0.0.0.0 --port 8080 --reload"
```

Comment:
We have this line in Dockerfile https://github.com/epam/badgerdoc/blob/main/annotation/Dockerfile#L20
Just specify build target
```yml
    build:
      context: .
      target: build-dev
```

Reply:
And do you think we need to mount the volume? We copy all necessary files during build in the Dockerfile

Reply:
Done